fdbserver/TLogServer.actor.cpp

```diff
@@ -2430,6 +2514,8 @@ ACTOR Future<Void> tLogCommit(TLogData* self,
 		qe.knownCommittedVersion = logData->knownCommittedVersion;
 		qe.messages = req.messages;
 		qe.id = logData->logId;
+		qe.prevVersion = req.seqPrevVersion;
+		qe.tLogLocIds = req.tLogLocIds;
```

Comment:
Here we write the new disk queue entries (prevVersion and tLogLocIds) even when version vector is disabled, which makes sense because the disk format should be the same with and without the feature enabled. We do not populate `tLogLocIds` when version vector is disabled, though. If the version vector knob is changed from disabled->enabled on a machine and during recovery encounter a  `tLogLocIds` that is empty for a version, will that version not be recovered when it should have been?